packages/table/preview/index.tsx

```diff
@@ -40,51 +40,52 @@ import {
     Utils,
 } from "../src/index";
 
+// tslint:disable:max-classes-per-file no-console jsx-no-lambda jsx-no-multiline-js
```

Comment:
same but would prefer not to touch this file too much. most of what you see here was done automatically by TSLint or VSCode formattin.